Source/GameBaseFramework/Private/Characters/GBFSimpleCharacterWithAbilities.cpp

```diff
@@ -60,4 +60,8 @@ void AGBFSimpleCharacterWithAbilities::BeginPlay()
 void AGBFSimpleCharacterWithAbilities::GetOwnedGameplayTags( FGameplayTagContainer & tag_container ) const
 {
     tag_container.AppendTags( StaticTags );
+}
+const TArray< TObjectPtr< const UGBFAbilitySet > > & AGBFSimpleCharacterWithAbilities::GetAbilitySets() const
```

Comment:
FORCEINLINE

Reply:
I thought forceinline was to be used for performance-critical functions. This is not the case here because we only call it in the editor and when validating ?

Reply:
(I'm saying that because of course i thought of using forceinline :P)

Reply:
yes, but since it's just a getter, why not?
it stays consistent with our other code.
And if it's called in lots of other places in the future, it will have an impact

Reply:
Done. (inlined)